File: Programmers/JAVA/lv1/lv1_없는 숫자 더하기.java
https://programmers.co.kr/learn/courses/30/lessons/86051?language=java

class Solution {
    public int solution(int[] numbers) {
        int answer = -1;
        int sum = 45;
        
        for(int i : numbers){
            sum -= i;
        }
        answer = sum;
        
        return answer;
    }
}

File: Programmers/C#/lv1/lv1_없는 숫자 더하기.cs
https://programmers.co.kr/learn/courses/30/lessons/86051

using System;

public class Solution
{
    public int solution(int[] numbers)
    {
        int answer = -1;
        int sum = 45;

        for (int i = 0; i < numbers.Length; i++)
        {
            sum -= numbers[i];
        }
        answer = sum;

        return answer;
    }
}